Automattic/jetpack
# Media Picker: Constrain preview image

The preview image in the media picker could sometimes break out of the container when the thumbnail image was used. This is because it's square.

## Proposed changes:
 This is a simple fix that adds width: auto to the image.

### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:
* Add the Jetpack Social Advanced plan to a test site
* Enable Social Image Generator in the editor sidebar for a post
* Select 'custom image'
* Use an image less than the 'large' size (usually 1024px)
* Without this change the square image will not be properly constrained
* With this change it should be constrained with a grey border

#### Before
<img width="279" alt="Screenshot 2023-06-20 at 13 42 39" src="https://github.com/Automattic/jetpack/assets/96462/8d26b045-30ae-4515-9b38-74f8248ece21">

#### After
<img width="279" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/036170dc-c8f6-4592-bec0-19dfca1c7543">
